### ISSUE TITLE
I get a false positive because I have luckypatcher installed

### ISSUE THREAD

**Issue by maxpilotto** [scottyab/rootbeer#94]
Shouldn't the `detectPotentiallyDangerousApps()` method be removed from the `isRooted()` check since luckypatcher can be used even without root access?

**Comment by scottyab** [comment]
Fair point, it _may_ be potentially dangerous but agree doesn't make sense for this to be in the primary `isRooted()` call. PR welcome.
